src/ui/MessageInput/index.jsx

```diff
@@ -97,7 +98,13 @@ const MessageInput = React.forwardRef((props, ref) => {
     setMentionedUsers,
   } = props;
   const textFieldId = messageFieldId || TEXT_FIELD_ID;
-  const { stringSet } = useContext(LocalizationContext);
+  const { stringSet } = useLocalization();
+  const { config } = useSendbirdStateContext();
+
+  const isFileUploadEnabled = channel?.channelType === 'open'
```

Comment:
Q. Is there a better way to get channelType by not forcing to pass `channel` prop to this component?

Reply:
So there should be another way to get a channel type instead of forcing to pass `channel` data to this component. @sravan-s will provide some utils to get this. (thx in advance 😁)